enrollment-server-onboarding-provider-innovatrics/src/test/java/com/wultra/app/onboardingserver/provider/innovatrics/InnovatricsRestApiServiceTest.java

```diff
@@ -51,7 +51,8 @@
 @ActiveProfiles("test")
 class InnovatricsRestApiServiceTest {
 
-    static final int PORT = 52936;
+    // TODO (racansky, 2024-06-17) find the way how to set the same random port for mock server and property
+    static final int PORT = 52937;
```

Comment:
It used to be in collision with `ZenidRestApiServiceTest`. Just a quick fix. Random ports will be handled later on.